e2e_samples/parking_sensors/src/ddo_transform/.devcontainer/Dockerfile

```diff
@@ -35,8 +43,8 @@ RUN apt-get autoremove -y \
     && rm -rf /var/lib/apt/lists/*
 ENV DEBIAN_FRONTEND=dialog
 
-# Set PACKAGE_VERSION to localdev
+# Set PACKAGE_VERSION to a PEP 440 compatible version (https://pep440.readthedocs.io/en/latest/)
```

Comment:
I'm getting a 404 Project not found on that link - this is going to hit on the link checks

Reply:
removed as unnecessary